code/Model/SiteTree.php

```diff
@@ -2036,12 +2033,6 @@ public function getCMSFields() {
 				. "</p>"), "Title");
 		}
 
-		// Backwards compat: Rewrite nested "Content" tabs to toplevel
```

Comment:
We haven't thrown deprecation notices on this, so might catch some devs out - but we've mentioned it in the 3.0.0 changelog: https://github.com/silverstripe/silverstripe-framework/blob/master/docs/en/04_Changelogs/3.0.0.md#changed-tab-paths-in-sitetree-getcmsfields-tab-paths